foundation/auth/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "google-cloud-auth"
-version = "0.9.2"
+version = "0.9.3"
```

Comment:
I would like to increase the minor version due to a change in the external interface. Could you please make the next version 0.10.0?

Reply:
done! does changing this version automatically release a new package on crates.io?

Reply:
Thanks.

> does changing this version automatically release a new package on crates.io?

No, we will test it in a integration test environment for a while and then release it manually.

Reply:
i saw that ci failed. i updated the dev-dependencies.